app/views/multi_transfers/steps/_set_params.html.erb

```diff
@@ -0,0 +1,31 @@
+<div class="alert alert-info" role="alert"><%= t('multi_transfers.step.set_params.description') %></div>
+
+<div class="form-inputs">
+  <%= simple_fields_for :transfer do |f| %>
+    <%= f.input :hours,
+      as: :integer,
+      input_html: {
+        value: @transfer_hours,
+        min: 0,
+        max: 20,
+        "data-rule-either-hours-minutes-informed" => "true",
+        "data-rule-range" => "[0,20]"
+      } %>
+    <%= f.input :minutes,
+      as: :integer,
+      input_html: {
+        value: @transfer_minutes,
+        min: 0,
+        max: 59,
+        step: 15,
+        "data-rule-either-hours-minutes-informed" => "true",
+        "data-rule-range" => "[0,59]"
+      } %>
+    <%= f.input :amount, as: :hidden, input_html: { value: @transfer_amount } %>
+    <%= f.input :reason, input_html: { value: @transfer_reason } %>
+
+    <% if @type_of_transfer.to_sym == :many_to_one && @target_accountable && @target_accountable.offers.length > 0 %>
```

Comment:
this condition seems a bit complex for a view, maybe we can move it to a helper: `if multitransfer_display_post?`

Reply:
moved to the controller